contracts/modules/cover/Quotation.sol

```diff
@@ -69,44 +69,73 @@ contract Quotation is Iupgradable {
   }
 
   /**
-   * @dev Expires a cover after a set period of time.
-   * Changes the status of the Cover and reduces the current
-   * sum assured of all areas in which the quotation lies
-   * Unlocks the CN tokens of the cover. Updates the Total Sum Assured value.
-   * @param _cid Cover Id.
+   * @dev Expires a cover after a set period of time and changes the status of the cover
+   * @dev Reduces the total and contract sum assured
+   * @param coverId Cover Id.
    */
-  function expireCover(uint _cid) public {
-    require(checkCoverExpired(_cid) && qd.getCoverStatusNo(_cid) != uint(QuotationData.CoverStatus.CoverExpired));
+  function expireCover(uint coverId) external {
 
-    tf.unlockCN(_cid);
-    bytes4 curr;
-    address scAddress;
-    uint sumAssured;
-    (,, scAddress, curr, sumAssured,) = qd.getCoverDetailsByCoverID1(_cid);
-    if (qd.getCoverStatusNo(_cid) != uint(QuotationData.CoverStatus.ClaimAccepted))
-      _removeSAFromCSA(_cid, sumAssured);
-    qd.changeCoverStatusNo(_cid, uint8(QuotationData.CoverStatus.CoverExpired));
+    uint expirationDate = qd.getValidityOfCover(coverId);
+    require(expirationDate < now, "Quotation: cover is not due to expire");
+
+    uint coverStatus = qd.getCoverStatusNo(coverId);
+    require(coverStatus != uint(QuotationData.CoverStatus.CoverExpired), "Quotation: cover already expired");
+
+    (/* claim count */, bool hasOpenClaim, /* accepted */) = tc.coverInfo(coverId);
+    require(!hasOpenClaim, "Quotation: cover has an open claim");
+
+    if (coverStatus != uint(QuotationData.CoverStatus.ClaimAccepted)) {
+      (,, address contractAddress, bytes4 currency, uint amount,) = qd.getCoverDetailsByCoverID1(coverId);
+      qd.subFromTotalSumAssured(currency, amount);
```

Comment:
Total sum assured is not adjusted if the claim payout did not succeed for too many times (status12).

Reply:
won't fix for now, we have an issue https://nexus-mutual.atlassian.net/browse/NM-176